Program.cs

```diff
@@ -722,6 +722,97 @@ public static void Props()
             Console.Clear();
         }
 
+        //User Powerup and prep
+        static void PowerUp()
+        {
+            Console.WriteLine("=========================================");
+            Console.WriteLine("==  Successful survived from the dark  ==");
+            Console.WriteLine("=========================================");
+            temp = Console.ReadLine();
+
+            while (true)
```

Comment:
There is currently no way of getting out of this while loop. Recommend either adding a bool variable that can be changed to false in it or simply removing the loop.